### ISSUE TITLE
[feat] Add Risc-v and Loongarch support

### ISSUE THREAD

**Issue by jiangcuo** [tauri-apps/tauri#8326]
### Describe the problem

Risc-v and Loongarch both support rust. 

### Describe the solution you'd like

Whether to consider adding support for these two architectures

**Comment by JonasKruckenberg** [comment]
Have you tried building an app for e.g. riscv linux?

I can think of a couple apis that would break, but for the most part it should just work™️

Maybe you can help us build out of list of issues that arise so were not completely in the dark

**Comment by jiangcuo** [comment]
@JonasKruckenberg 
Hi , When I use `npm run tauri dev`

<img width="1256" alt="image" src="https://github.com/tauri-apps/tauri/assets/49061187/c360903b-0b12-4f84-b9cd-9954b3632b45">

I opened index.js. Found that loongarch64 and riscv64 were not added.

I had to try use `cargo tauri dev` with `tauri.conf.json`
```json
 "build": {
    "beforeDevCommand": "npm run dev",
    "beforeBuildCommand": "npm run build",
    "devPath": "http://localhost:1420",
    "distDir": "../dist",
    "withGlobalTauri": false
  }
```
`cargo tauri dev`  is ok. I can access my app.But  when i use `cargo tauri build` ,I will revice error :
```
Finished release [optimized] target(s) in 36.62s
Error failed to build bundler settings: Unable to determine target-architecture
```
Manually run targer/release/binfile, I can open the app, but the program enter white screen.

<img width="968" alt="image" src="https://github.com/tauri-apps/tauri/assets/49061187/3d05e88d-4161-4547-a698-cde2a5b5917d">

It looks like the front-end service isn't working properly

I checked this issue and the problem is still there
https://github.com/tauri-apps/tauri/issues/5143

Note,I'm testing in VM with virtio-gpu。
I'm going to test with virtio-gpu-gl or a physical machine to rule out opengl issues

Best Regards
